test.py

```
@@ -1,5 +1,40 @@
+    return 3.14 * r * h * h
+
+def main():
+    result = do_something()
+    print("Done processing")
+    for i in result:
+        if i > 100:
+            print("Large value:", i)
+        else:
+            print("Small value:", i)
+    try:
+        val = int("abc")
```

Ruff (0.12.2)

Local variable `val` is assigned to but never used

Remove assignment to unused variable `val`

(F841)